src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import App from './App';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <>
+    <App />
+  </>
+);
```

Comment:
I suggest to wrap <App /> in <React.StrictMode> as below:
`root.render(
  <React.StrictMode>
    <App />
  <React.StrictMode/>
);`